content/desktop/release-notes.md

```diff
@@ -24,6 +24,64 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
+## 4.27.0
+
+{{< release-date date="2023-12-14" >}}
+
+{{< desktop-install all=true version="4.27.0" build_path="/TBD/" >}}
+
+### New
+
+- Docker init now supports Java and is generally available to all users (GA)
+- Synchronized File Shares are designed to provide fast and flexible host-to-VM file sharing within Docker Desktop. Utilizing the technology behind [Docker’s acquisition of Mutagen](https://www.docker.com/blog/mutagen-acquisition/), this feature provides an alternative to virtual bind mounts that uses synchronized filesystem caches, improving performance for developers working with large codebases.
```

Comment:
Should this link to the documentation at https://docs.docker.com/desktop/synchronized-file-sharing/